skorch/tests/test_probabilistic.py

```diff
@@ -12,12 +12,23 @@
 import torch
 from torch.testing import assert_allclose
 
+from skorch._version import Version
 from skorch.utils import is_torch_data_type
 from skorch.utils import to_numpy
 
 
 gpytorch = pytest.importorskip('gpytorch')
 
+# check that torch version is sufficiently high for gpytorch, otherwise skip
+version_gpytorch = Version(gpytorch.__version__)
+version_torch = Version(torch.__version__)
+# TODO: remove if newer GPyTorch versions are released that no longer require
+# the check.
+if (version_gpytorch >= Version('1.9')) and (version_torch < Version('1.11')):
```

Comment:
The alternative way to skip a whole module is setting [pytestmark](https://docs.pytest.org/en/7.1.x/how-to/skipping.html#skip-all-test-functions-of-a-class-or-module) to `pytest.mark.skipif`. I have no strong opinion either way.

Reply:
I didn't know that, thanks for the pointer. Between the two variants, I think the existing one is a bit more obvious without consulting the pytest docs.